src/Symfony/Component/Clock/Tests/MockClockTest.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Clock\Tests;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Clock\MockClock;
+
+class MockClockTest extends TestCase
+{
+    public function testConstruct()
+    {
+        $clock = new MockClock();
+        $this->assertSame('UTC', $clock->now()->getTimezone()->getName());
+
+        $tz = date_default_timezone_get();
+        $clock = new MockClock($tz);
+        $this->assertSame($tz, $clock->now()->getTimezone()->getName());
```

Comment:
These tests unnecessarily depend on the global timezone settings. Let's decouple them.

```suggestion
        $clock = new MockClock('Europe/Paris');
        $this->assertSame('Europe/Paris', $clock->now()->getTimezone()->getName());
```

Reply:
updated